_config.php

```diff
@@ -19,3 +19,9 @@
     //using the same db relations for the versioned documents, as for the actual documents
     $config->update('DMSDocument_versions', 'db', $config->get('DMSDocument', 'db'));
 }
+
+// add dmsassets folder to file system sync exclusion
+if (strpos($config->get('DMS', 'folder_name'), 'assets/') === 0) {
+    $folderName = str_replace('assets/', '', $config->get('DMS', 'folder_name'));
```

Comment:
Maybe we could do `substr($config->get('DMS', 'folder_name'), 0, 7)` instead? I have a feeling this wouldn't work if the folder name was a subfolder like "assets/my-awesome-assets/some-other-folder" for example